rocks.kfs.StepsToCare/Properties/AssemblyInfo.cs

```diff
@@ -22,10 +22,10 @@
 // associated with an assembly.
 [assembly: AssemblyTitle( "rocks.kfs.StepsToCare" )]
 [assembly: AssemblyProduct( "rocks.kfs.StepsToCare" )]
-[assembly: AssemblyCopyright( "Copyright © Kingdom First Solutions 2024" )]
+[assembly: AssemblyCopyright( "Copyright © Kingdom First Solutions 2025" )]
 
 // Auto increment assembly versions
-[assembly: AssemblyVersion( "2.0.1.*" )]
+[assembly: AssemblyVersion( "2.0.2.*" )]
```

Comment:
I forgot to actually push my updated version last time, so this will actually be 2.0.3 to match what the Rock Shop will have. 
```suggestion
[assembly: AssemblyVersion( "2.0.3.*" )]
```